tlp-pcilist

```diff
@@ -58,9 +62,9 @@ foreach (`lspci -m`) {
         # get Runtime PM mode
         my $pmode = catsysf ("$devc");
         # output pm mode and device data
-        printf "%s/power/control = %-4s (%s, %s, %s)\n", $devp, $pmode, $class, $classdesc, $driver;
+        printf "%s/power/control = %-4s  %s/power/runtime_status = %-4s (%s, %s, %s)\n", $devp, $pmode, $devp, $runtime_status, $class, $classdesc, $driver;
```

Comment:
Including the full path of runtime status makes the output much wider.
Format `%-4s` breaks the column alignment because runtime_status is up to 9 chars wide.

I suggest to change to
`"%s/power/control = %-4s, runtime_status = %-9s`